src/main/java/org/springframework/data/repository/config/RepositoryConfigurationSourceSupport.java

```diff
@@ -28,26 +28,32 @@
 
 /**
  * Base class to implement {@link RepositoryConfigurationSource}s.
- * 
+ *
  * @author Oliver Gierke
  * @author Thomas Darimont
  * @author Peter Rietzler
+ * @author Jens Schauder
  */
 public abstract class RepositoryConfigurationSourceSupport implements RepositoryConfigurationSource {
 
 	protected static final String DEFAULT_REPOSITORY_IMPL_POSTFIX = "Impl";
 
 	private final Environment environment;
 
+	private final RepositoryBeanNameGenerator beanNameGenerator;
+
 	/**
 	 * Creates a new {@link RepositoryConfigurationSourceSupport} with the given environment.
-	 * 
+	 *
 	 * @param environment must not be {@literal null}.
+	 * @param classLoader
 	 */
-	public RepositoryConfigurationSourceSupport(Environment environment) {
+	public RepositoryConfigurationSourceSupport(Environment environment, ClassLoader classLoader) {
 
 		Assert.notNull(environment, "Environment must not be null!");
```

Comment:
I'd rather put a blank line after the assert and remove the one below to group the assignments.